client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -196,32 +223,36 @@ function ReaderSubscriptionListItem( {
 }
 
 export default compose(
-	connect( ( state, ownProps ) => {
-		const feed = getFeed( state, ownProps.feedId );
+	connect(
+		( state, ownProps ) => {
+			const feed = getFeed( state, ownProps.feedId );
 
-		if ( feed ) {
-			const follow = getReaderFollowForFeed( state, parseInt( ownProps.feedId ) );
+			if ( feed ) {
+				const follow = getReaderFollowForFeed( state, parseInt( ownProps.feedId ) );
 
-			if ( follow ) {
-				// Add site icon to feed object so have icon for external feeds when not set
-				if ( feed.site_icon === undefined ) {
-					feed.site_icon = follow.site_icon;
-				}
-				// Add date_subscribed timestamp to feed object when not set
-				if ( feed.date_subscribed === undefined || isNaN( feed.date_subscribed ) ) {
-					feed.date_subscribed = follow.date_subscribed;
-				}
-				// Add last_update timestamp to feed object when not set
-				if ( feed.last_update === undefined || isNaN( feed.last_update ) ) {
-					feed.last_update = follow.last_updated;
+				if ( follow ) {
+					// Add site icon to feed object so have icon for external feeds when not set
+					if ( feed.site_icon === undefined ) {
+						feed.site_icon = follow.site_icon;
+					}
+					// Add date_subscribed timestamp to feed object when not set
+					if ( feed.date_subscribed === undefined || isNaN( feed.date_subscribed ) ) {
+						feed.date_subscribed = follow.date_subscribed;
+					}
+					// Add last_update timestamp to feed object when not set
+					if ( feed.last_update === undefined || isNaN( feed.last_update ) ) {
+						feed.last_update = follow.last_updated;
+					}
 				}
 			}
-		}
 
-		return {
-			feed,
-		};
-	} ),
+			return {
+				feed,
+				isLoggedIn: isUserLoggedIn( state ),
+			};
+		},
+		{ registerLastActionRequiresLogin }
```

Comment:
alternative to my note above in destructure, we could also avoid the shadowing here by putting this under a new property name.

Reply:
I didn't need to change this.